nx.json

```diff
@@ -0,0 +1,37 @@
+{
+  "extends": "nx/presets/npm.json",
+  "tasksRunnerOptions": {
+    "default": {
+      "runner": "@nrwl/nx-cloud",
+      "options": {
+        "cacheableOperations": [
+          "build",
+          "cs-check",
+          "cs-format",
+          "lint",
+          "test",
+          "test:update",
+          "test:watch",
+          "precommit",
+          "publish-to-npm",
+          "test-coverage",
+          "build:lib",
+          "prepublishOnly",
+          "build:dist",
+          "publish-to-gh-pages"
+        ],
+        "accessToken": "ZmFmMjBmY2QtZTc1Mi00MGViLThkMGItODA5OTU4MDc5MWEyfHJlYWQtd3JpdGU="
```

Comment:
Is this a secret that needs to be passed in as a variable? Also, I'm not sure we want to use the cloud... and if we did, we really don't want this secret sitting in a public file (we have a place for them). Can we not use the cloud services by default?

Reply:
The cloud caches the build and in later builds, only the parts that are affected are built. Since it's free for open-source I think we can make use of it.
Though you're correct. See [here](https://nx.dev/nx-cloud/account/access-tokens) for access tokens. To pass the access token as an environment variable for CI workflow, you can set NX_CLOUD_ACCESS_TOKEN in secrets. But it doesn't matter at the moment because only a few people have write access to the source code. But please consider creating an Nx Cloud account and passing the token so that we can make use of it.
If you're still not sure, you can remove the token and it will disable the cloud for the time being.
Best regards

Reply:
@ahkhanjani can you please remove the token for now. I've created an issue #3005 to make sure we get a proper access token setup for our builds!